library/reminder/src/BirthdayReminder.php

```diff
@@ -57,6 +57,11 @@ public function isDisplayBirthdayAlert()
         return false;
     }
 
+    public static function __callStatic($name, $arguments)
+    {
+        // TODO: Implement __callStatic() method.
```

Comment:
Mistake :)

Reply:
no prob :)
Just brought in your fix. thanks!